first_models.tex

```diff
@@ -53,9 +53,35 @@ \subsection{Bag of Words}
 \label{tab:bow_cw}
 \end{table}
 
-\subsection{Forest}
+\subsection{Random Forest}
 \label{sec:forest}
 
+Based on impressive successes in software author recognition using random forest
+methods \cite{c3}, there was interest in the accuracy of a random forest
+predictor, even with its unsuitability for use in a competition judged by
+log-loss. Initially the forest was fed encoded sentences directly.
```

Comment:
is it worth explaining why log-loss is bad for random forest? I'm more asking this because I myself don't know why it's bad.